### PR TITLE
Added OK alarms to send notification to Slack

### PR DESCRIPTION
**Pull request by bryan-robitaille**
This PR adds the functionality to alert the #exposure-healthcare-dev Slack channel when certain alarms reset.  This ensures that after-hours non-urgent alarms/alerts can be quickly determined to either be resolved or requiring the need  for further investigation.

Alarms that now have both an Alarm and Ok status:
- Portal CPU Utilization
- Portal Memory Utilization
- Abnormal Response Times (Anomaly Detection at 3 Standard Deviations)
- Abnormal Service Availability (Anomaly Detection at 3 Standard Deviations)

### CODE REVIEW

File: config/terraform/aws/lambda/lambda_notify_slack.js [hunk 1]
```diff
@@ -29,21 +31,42 @@ exports.handler = function(event, context) {
...
     
-    for(var dangerMessagesItem in dangerMessages) {
-        if (message.indexOf(dangerMessages[dangerMessagesItem]) != -1) {
-            severity = "danger";
-            postData.icon_emoji = ":rotating_light:";
+    for(var okMessagesItem in okMessages) {
+        if (message.indexOf(okMessages[okMessagesItem]) != -1) {
+            message = "Alarm Status now OK - " + getMessage(message); 
+            needs_processing = false;
             break;
```
Comment (pcraig3) [inline config/terraform/aws/lambda/lambda_notify_slack.js:44]: No `severity` or `icon_emoji`?

Comment (bryan-robitaille) [reply config/terraform/aws/lambda/lambda_notify_slack.js:44]: The default emoi is set on line 19 as being the loadspeaker.
https://github.com/cds-snc/covid-alert-portal/blob/f01c18c5f70267bc7084293927b3eec977c155cc/config/terraform/aws/lambda/lambda_notify_slack.js#L19
The default severity is also set on line 24.
https://github.com/cds-snc/covid-alert-portal/blob/f01c18c5f70267bc7084293927b3eec977c155cc/config/terraform/aws/lambda/lambda_notify_slack.js#L24

Basically by default everything is good until it's not.

**Review by pcraig3 — APPROVED** [review]
Approved!
